tomviz/python/tomviz/io/ser.py

```diff
@@ -0,0 +1,811 @@
+"""
+This module provides an interface to the SER file format written by FEI
+and Therm0 Fischer's program TIA.
+
+It reads STEM and TEM images and other datasets.
+
+It is based on information provided by Dr Chris Boothroyd and work by
+Peter Ercius' original serReader code in Matlab.
+
+Note
+----
+General users:
+    Use the simplified ser.serReader() function to load the data and meta
+    data as a python dictionary.
+
+Advanced users and developers:
+    Access the file internals through the ser.fileSER() class.
+
+This function is based on the Python code by Peter Ercius,
+NCEM - Berkeley National Lab.
+openNCEM - https://github.com/ercius/openNCEM
```

Comment:
Again, GPL code from my inspection, relicense?

Reply:
Sorry, I misread - I see from the text in the README it is dual licensed GPL or MIT. We can take MIT fine!